client/src/components/EditProfile/EditPhoto/useStyles.ts

```diff
@@ -16,6 +17,14 @@ const useStyles = makeStyles((theme) => ({
   reminder: {
     padding: theme.spacing(2.5),
     width: '16vw',
+    color: '#ff0000',
+    animation: '$colorAnimate 10s infinite',
+  },
+  '@keyframes colorAnimate': {
```

Comment:
You can stick to using the primary/secondary colours of the app :)

Reply:
Done.